.github/actions/setup-wp-env/action.yml

```
@@ -0,0 +1,53 @@
+name: Setup WP Env
+description: Setup WordPress environment for testing
+
+inputs:
+    wp:
+        description: WordPress version
+        required: true
+    php:
+        description: PHP version
+        required: true
+    plugins:
+        description: WordPress plugins
```

```suggestion
        description: A comma separated list of plugins to install (e.g. `elementor, elementor-pro`)
```

also others?
should it be paths? names?

---

Done